main.go

```diff
@@ -34,7 +34,7 @@ func setupRouter() *gin.Engine {
 	v1.GET("/version", routes.VersionRoute(Version))
 
 	v1.PUT("/er", routes.AuthMiddleware(), routes.CreateEmergencyRoom)
-	v1.GET("/er", routes.GetEmergencyRooms)
+	v1.GET("/er/ger/:page", routes.GetEmergencyRooms)
```

Comment:
This might be personal preference, but I thing page should be a query parameter, not a path parameter.
Why use "ger" instead of "er"?
Anyhow please also generate new swagger documentation using "swag init" as per README